test/e2e/deployment.go

```diff
@@ -348,114 +345,34 @@ func testRollingUpdateDeployment(f *framework.Framework) {
 	Expect(len(allOldRSs[0].Spec.Template.Labels[extensions.DefaultDeploymentUniqueLabelKey])).Should(BeNumerically(">", 0))
 }
 
-func testRollingUpdateDeploymentEvents(f *framework.Framework) {
```

Comment:
I don't think we guarantee scaling direction for rolling deployments. I can move 2. to another test.